docs/reference/snapshot-restore/monitor-snapshot-restore.asciidoc

```diff
@@ -149,8 +149,7 @@ detailed insight into the current state of the recovery process by using <<indic
 [discrete]
 [[get-snapshot-stop-snapshot]]
 === Stop snapshot and restore operations
-The snapshot and restore framework allows running only one snapshot or one restore operation at a time. If a currently
-running snapshot was started by mistake, or takes unusually long, it can be stopped using the <<delete-snapshot-api,delete snapshot API>>.
+If a currently running snapshot was started by mistake, or takes unusually long, it can be stopped using the <<delete-snapshot-api,delete snapshot API>>.
```

Comment:
```suggestion
To stop a currently running snapshot that was started by mistake or is taking unusually long, use
the <<delete-snapshot-api,delete snapshot API>>.
```

Reply:
Suggest reordering this sentence to make it more focused on the action. I realize that I missed that change in the previous PR 😆